deps/rabbit/test/quorum_queue_SUITE.erl

```diff
@@ -2526,6 +2527,23 @@ consumer_metrics(Config) ->
                                                         Key == consumers
                                                 end, PropList)).
 
+up_metrics(Config) ->
+    [Server | _] = rabbit_ct_broker_helpers:get_node_configs(Config, nodename),
+
+    Ch1 = rabbit_ct_client_helpers:open_channel(Config, Server),
+    QQ = ?config(queue_name, Config),
+    ?assertEqual({'queue.declare_ok', QQ, 0, 0},
+                 declare(Ch1, QQ, [{<<"x-queue-type">>, longstr, <<"quorum">>}])),
+
+    RaName = ra_name(QQ),
+    {ok, _, {_, Leader}} = ra:members({RaName, Server}),
+    timer:sleep(5000),
+    QNameRes = rabbit_misc:r(<<"/">>, queue, QQ),
+    [{_, PropList, _}] = rpc:call(Leader, ets, lookup, [queue_metrics, QNameRes]),
```

Comment:
Converting this proplist to a map and getting the key would be clearer.

Reply:
I copied that from the surrounding test cases. I'm happy to change it in this test case and the surrounding test cases as well. Thanks.

Reply:
Using `proplists:get_value` to make it clearer.